mbuild/compound.py

```diff
@@ -389,14 +403,14 @@ def rigid_particles(self, rigid_id=None):
             if particle.rigid_id is not None:
                 yield particle
 
-    def set_rigid(self, name=False, rigid_id=None):
+    def set_rigid(self, particle_name=False, rigid_id=None):
         """Treat the Compound and all successors as a single rigid body.
 
         Provides a unique rigid body ID to a Compound and its successors.
-        If 'name' is specified, only sucessors with this name are provided
-        with a rigid_id.  This is useful for Compounds where only some of
-        the particles should be rigid (such as a tethered nanoparticle with
-        a rigid core).
+        If 'particle_name' is specified, only sucessors with this name are
```

Comment:
The current code isn't technically limited to particles (the bottom of the hierarchy), right? We want to remain consistent with nomenclature.